src/widget_builder.js

```diff
@@ -226,6 +227,26 @@ const buildAgGrid = (view, gridData, gridOptions_str, div, sheet, dropdownMulti
     if (view.model.get('sync_grid')) {
         exportFunc.exportGrid(gridOptions, view);
     }
+
+    // Listen to scroll property changes from python side
+    view.model.on('change:scroll', () => {
+        let scroll = view.model.get('scroll');
+        let gridBody = view.gridDiv.querySelector("div[ref=eBodyViewport]");
+        gridBody.scrollLeft = scroll[0]
+        gridBody.scrollTop = scroll[1]
+    });
+
+
+    const saveScroll = Helpers.debounce((left, top) => {
+        console.log(`bodyScroll(${left}, ${top})`);
+        view.model.set('scroll', [left, top]);
+        view.touch();
+    }, 300);
+
+    // Listen to scroll changes from the UI
+    gridOptions.api.addEventListener('bodyScroll', params => {
+        saveScroll(params.left, params.top);
```

Comment:
saveScroll - can you find a better name in their convention?

Reply:
the only convention I could find was to use the word `export` when setting property for the backend.
changed to `exportScroll`.